$OpenBSD$

SPDX-License-Identifier: BSD-3-Clause

Relax the ghc-prim upper bound so this aeson fork builds with GHC 9.10+,
and allow the fixed text-iso8601 version for HSEC-2026-0007.

Index: aeson.cabal
--- aeson.cabal.orig
+++ aeson.cabal
@@ -100,7 +100,7 @@
     , containers        >=0.5.10.2 && <0.7
     , deepseq           >=1.4.3.0  && <1.6
     , exceptions        >=0.10.4   && <0.11
-    , ghc-prim          >=0.5.0.0  && <0.12
+    , ghc-prim          >=0.5.0.0  && <0.14
     , template-haskell  >=2.12.0.0 && <2.22
     , text              >=1.2.3.0  && <1.3  || >=2.0 && <2.2
     , time              >=1.8.0.2  && <1.13
@@ -132,7 +132,7 @@
     , semialign             >=1.3      && <1.4
     , strict                >=0.5      && <0.6
     , tagged                >=0.8.7    && <0.9
-    , text-iso8601          >=0.1      && <0.2
+    , text-iso8601          >=0.1      && <0.3
     , text-short            >=0.1.5    && <0.2
     , th-abstraction        >=0.5.0.0  && <0.7
     , these                 >=1.2      && <1.3
